library/src/main/java/com/novoda/downloadmanager/DownloadBatchFactory.java

```diff
@@ -68,13 +67,15 @@ static DownloadBatch newInstance(Batch batch,
             downloadFiles.add(downloadFile);
         }
 
+        DownloadBatchStatus.Status status = DownloadBatchStatus.Status.UNKNOWN;
```

Comment:
What makes this status UNKNOWN instead of DOWNLOAD_ERROR?

Reply:
It's UNKNOWN because it's just being created, so the status is not defined yet. It will be added to queue in the downloader later on